### ISSUE TITLE
is it possible to use groq (llama 3.1 70b) api to compress text instead of running your own model?

### ISSUE THREAD

**Issue by sprappcom** [AlexBuz/llama-zip#20]
much prefer if i can use the ai apis available online for the compress/decompress instead of self hosting. is this possible?

**Comment by AlexBuz** [comment]
Unfortunately, this does not appear to be possible at the moment, as the Groq API [does not provide access to the model's logits](https://console.groq.com/docs/openai) during inference, which would be necessary for compression and decompression.

**Comment by sprappcom** [comment]
@AlexBuz isnt the llama models the same? e.g. llama 3.1 8b?

**Comment by AlexBuz** [comment]
Groq does not provide access to the logits during inference, no matter which model is used, if that's what you're asking.
